.sample.env

```diff
@@ -1,4 +1,4 @@
 ETHEREUM_CLIENT_CLASS="AlchemyClient"
 ETHEREUM_CLIENT_API_KEY="YOUR KEY"
-ETHEREUM_NETWORK="eth-mainnet"
```

Comment:
Deleting this may cause an error when running the command `bundle exec clockwork config/main_importer_clock.rb`.
ETHEREUM_NETWORK is used in /ethscriptions-indexer/app/models/eth_block.rb:33
![review](https://github.com/0xFacet/ethscriptions-indexer/assets/117507973/ca8090e3-57c6-4d62-a609-0dcf4d7115d5)

Reply:
Nice catch! I intend to merge this shortly!